PINCache.podspec

```diff
@@ -22,11 +24,12 @@ EOS
   s.prefix_header_contents = pch_PIN
   s.subspec 'Core' do |sp|
       sp.source_files  = 'Source/*.{h,m}'
-      sp.dependency 'PINOperation', '~> 1.2.1'
+      sp.dependency 'PINOperation', '~> 1.2.2'
   end
   s.subspec 'Arc-exception-safe' do |sp|
       sp.dependency 'PINCache/Core'
       sp.source_files = 'Source/PINDiskCache.m'
       sp.compiler_flags = '-fobjc-arc-exceptions'
   end
+  s.resource_bundles = { 'PINCache' => ['Source/PrivacyInfo.xcprivacy'] }
```

Comment:
This is an important line. Adding a Privacy Manifest to resources would end up clobbering an app's Privacy Manifest. Popular open source libraries (Firebase, Alamofire, etc.) have been using this approach for several months with no issues.